CITATION.cff

```diff
@@ -0,0 +1,16 @@
+cff-version: 1.2.0
+message: "If you use this software, please cite it as below."
```

Comment:
Why "software"?

Reply:
@bianchini88  Good catch, What about "If you use content of this website, please cite as below."?

Reply:
@bedroesb Maybe something like "When you use the information provided on this website, please cite as below". Just to make sure it is about the information on the pages and not only the text itself.

Reply:
@bianchini88 not sure which ones is better if I ma honest. Other opinions? @elixir-europe/rdmkit-editors 

```
If you use content of this website, please cite as below
```
vs
```
When you use the information provided on this website, please cite as below
```

Reply:
I prefer latter "When you..", it's more welcoming :)

Reply:
I agree! And if it is:

```
When you use content of this website, please cite as below
```

Reply:
Sounds good to me! I see the difference between the use of "content of.." vs "information provided.." and I prefer former ("content of..") :)

Reply:
Just to clarify, I suggested moving away from "using content" because that could be interpreted as "if you are using the actual text" (i.e. if you quote) and that is probably not what we want. On the other hand, I do appreciate that the "content of" sentence flows a bit better. Another possible solution would be to move away from the word "use" entirely. Something like ``` When referencing this website, please cite as below ```. Anyway, I am probably overthinking this.

Reply:
I don't think you are overthinking, I appreciate your view @bianchini88  How about "When using content from the RDMkit pages, please reference the website by citing it as follows:" ?

Reply:
@Nazeeefa, it is a better phrasing, but the ambiguity in the word "using" is still there. We inherited this word from the initial commit, which contained a sentence that was software-related, and one does not "use" RDMkit in the same way one would "use" a piece of software. I think the most correct verb to substitute for "using" would be "referencing" in this case (but feel free to suggest some others). I would suggest ```When referencing content from the RDMkit pages, please cite as follows:```

Reply:
I like that version!